### PR TITLE
Add app deployment example

### PR DESCRIPTION
**Pull request by alexjh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,123 @@
...
+
+UAA_CA_CERT="$(kubectl get secret secret --namespace ${UAA_NAMESPACE} -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+CF_CA_CERT="$(kubectl get secret secret --namespace ${CF_NAMESPACE} -o jsonpath="{.data['internal-ca-cert']}" | base64 --decode -)"
+CF_PASSWORD="$(kubectl get secret secret --namespace ${CF_NAMESPACE} -o jsonpath="{.data['cluster-admin-password']}" | base64 --decode -)"
+
+helm install ./chart --name mysql-instance --namespace ${SIDECAR_NAMESPACE} \
+	--set "env.UAA_CA_CERT=${UAA_CA_CERT}" \
+	--set "env.CF_CA_CERT=${CF_CA_CERT}" \
+	--set "env.SERVICE_LOCATION=http://cf-usb-sidecar-mysql.${SIDECAR_NAMESPACE}.svc.cluster.local:8081" \
+	--set "env.SERVICE_MYSQL_HOST=AUTO" \
```
Comment (andreas-kupries) [inline README.md]: Should we document how to go about using non-`AUTO` ?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,123 @@
...
+docker login # if authorization is required
+```
+
+Then build the top level dependencies and the sidecar:
+
+```
+make tools
+make build-image
+cd csm-extensions/services/dev-mysql
+make build-image public-image helm
```
Comment (andreas-kupries) [inline README.md]: `publish`, not `public`.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,123 @@
...
+	--set "env.CF_CA_CERT=${CF_CA_CERT}" \
+	--set "env.SERVICE_LOCATION=http://cf-usb-sidecar-mysql.${SIDECAR_NAMESPACE}.svc.cluster.local:8081" \
+	--set "env.SERVICE_MYSQL_HOST=AUTO" \
+	--set "env.CF_ADMIN_USER=admin" \
+	--set "env.CF_ADMIN_PASSWORD=${CF_PASSWORD}" \
+	--set "env.CF_DOMAIN=${CF_DOMAIN}"
+```
+
+Eventually you should see two pods start in the `SIDECAR_NAMESPACE`:
+
```
Comment (andreas-kupries) [inline README.md]: There should be a third `setup` pod, running temporarily (errand).
Not sure if it makes sense to note that.
That said, if the `setup` errand fails not knowing that that pod was supposed to be there might be confusing.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,123 @@
...
+Once the pods are ready, it should be available in the marketplace:
+
+```
+$ cf marketplace
+Getting services from marketplace in org org / space space as admin...
+OK
+
+service    plans     description
+postgres   default   Default service
```
Comment (andreas-kupries) [inline README.md:163]: Should we mention postgres given that the example is focused on mysql ?

Comment (andreas-kupries) [reply README.md:163]: Oh, wait, the django example is also postgres based.

File: README.md [hunk 5]
```diff
@@ -1,13 +1,123 @@
...
+At this point, services can be made available to apps. In this case we're going to use the [django-cms](https://github.com/scf-samples/django-cms) app.
+
+```
+git clone https://github.com/scf-samples/django-cms -b scf
+cd django-cms
+cf create-service postgres default django-cms-db
+cf push --no-start django-cms
+cf set-env django-cms DISABLE_COLLECTSTATIC 1
+cf set-env django-cms DJANGO_SETTINGS_MODULE settings
+cf restart django-cms
```
Comment (andreas-kupries) [inline README.md]: `cf start` is good enough, given the `--no-start` argument to `push`.

**Review by andreas-kupries — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alexjh) [reply README.md]: Fixed in a109d80

File: README.md (hunk 3 above)
Comment (alexjh) [reply README.md]: Good point. Added a detail about that in 2d8ca8e.

File: README.md (hunk 5 above)
Comment (alexjh) [reply README.md]: Fixed in d2eace9

**Review by andreas-kupries — APPROVED** [review]
Looking good.

**Pull request merged**